unstable/wlr-layer-shell-unstable-v1.xml

```diff
@@ -127,14 +127,19 @@
 
     <request name="set_exclusive_zone">
       <description summary="configures the exclusive geometry of this surface">
-        Requests that the compositor avoids occluding an area of the surface
-        with other surfaces. The compositor's use of this information is
+        Requests that the compositor avoids occluding an area with other
+        surfaces. The compositor's use of this information is
         implementation-dependent - do not assume that this region will not
         actually be occluded.
 
-        A positive value is only meaningful if the surface is anchored to an
-        edge, rather than a corner. The zone is the number of surface-local
-        coordinates from the edge that is considered exclusive.
+        A positive value is only meaningful if the surface is anchored to one
+        edge or an edge and both perpendicular edges. If the surface is not
+        anchored, anchored to only two perpendicular edges (a corner), anchored
+        to only two parallel edges or anchored to all edges, a positive value
+        will be treated the same as zero.
```

Comment:
Maybe this should be a protocol error

Reply:
(Would require a version bump, and can be done separately)

Reply:
I don't have an opinion on this

Reply:
A protocol error would be nice for a future patch, agreed.

Reply:
Does that mean it will be required to set the exclusive zone to zero in the same commit as you change the anchor to one that does not allow an exclusive zone?

Reply:
Yeah, that would make sense